src/Helpers/babylon.particleHelper.ts

```diff
@@ -0,0 +1,156 @@
+module BABYLON {
+    /**
+     * Represents all the data needed to create a ParticleSystem.
+     */
+    export interface IParticleSystemData {
+        type: ParticleSystemType;
```

Comment:
We should also be able to define the emitter type (+ parameters)

Reply:
I'm not sure how to do this?
Can you give me an example please?

Reply:
It is not the emitter but the emitter shape. The shape that the particles will use